### ISSUE TITLE
Community Batch Imports: add links between "Pending Imports" and "New Imports" pages for easier navigation.

### ISSUE THREAD

**Issue by RayBB** [internetarchive/openlibrary#10154]
### Proposal

https://openlibrary.org/import/batch/pending and https://openlibrary.org/import/batch/new don't link to each other.
Lets add links between them at the bottom for easier access.

#### Requirements Checklist

* [ ]

#### Related files

* See https://github.com/internetarchive/openlibrary/issues/9938

#### Stakeholders

* @scottbarnes 

<hr>

#### Instructions for Contributors

Please [run these commands](https://github.com/internetarchive/openlibrary/wiki/Git-Cheat-Sheet#working-on-your-branch) to ensure your repository is up to date **before** [creating a new branch](https://github.com/internetarchive/openlibrary/wiki/Git-Cheat-Sheet#making-changes-and-creating-a-pull-request) to work on this issue and **each time after** pushing code to Github, because the pre-commit bot may add commits to your PRs upstream.

**Comment by phoenix7815** [comment]
Hello @RayBB , I would like to work on this.
Can you please assign me this?

**Comment by scottbarnes** [comment]
I assigned this to you, @phoenix7815. Please let us know if you run into any issues.

**Comment by ananyakaligal** [comment]
Hello @RayBB could you assign me this issue if @phoenix7815 isn't working on it?

**Comment by phoenix7815** [comment]
I am still working on this

**Comment by mekarpeles** [comment]
@phoenix7815 do you have any questions? Is this something you're actively working on?

See example here:
https://github.com/internetarchive/openlibrary/blob/master/openlibrary/macros/SearchNavigation.html#L6C7-L6C59

I'd create a new template called https://github.com/internetarchive/openlibrary/blob/master/openlibrary/templates/BatchImportNavigation.html

```
$ def with ()

<div class="batch-imports-nav">
  <a class="$('selected' if ctx.path.endswith('new') else '')" href="/import/batch/new">$_("Submit")</a>
  <a class="$('selected' if ctx.path.endswith('pending') href="/import/batch/pending">$_("Pending")</a>
</div>
```

Then I'd update the following two pages at these locations:
* https://github.com/internetarchive/openlibrary/blob/master/openlibrary/templates/batch_import_view.html#L5
* https://github.com/internetarchive/openlibrary/blob/master/openlibrary/templates/batch_import.html#L9

To include this new nav menu via:

```
$:templates.BatchImportNavigation()
```

**Comment by ananyakaligal** [comment]
@mekarpeles, this issue seems relatively straightforward. Could I take it on?

**Comment by RayBB** [comment]
@ananyakaligal you're assigned. Good luck :)

**Comment by RayBB** [comment]
I recommend following the steps in the documentation to fully reset your environment. If that doesn't work then please post the exactly error log here.

**Comment by ananyakaligal** [comment]
@RayBB alright, thankyou

**Comment by ananyakaligal** [comment]
Hello @RayBB , Here is the design for your review, I've designed it similar to the search navigation bar. If it's good to go, I'll proceed with making it functional, otherwise let me know if any changes are needed. 

![Screenshot from 2025-01-05 22-20-20](https://github.com/user-attachments/assets/7f4f77ed-2a9e-4798-9999-7142f6b8d8d3)
![Screenshot from 2025-01-05 22-23-23](https://github.com/user-attachments/assets/6107fd9b-4787-42ba-ac44-a82afb06529e)

**Comment by RayBB** [comment]
That looks nice to me. Sre you using Meks template for that mockup? 
In any case go ahead and open a PR :) 
@ananyakaligal

**Comment by ananyakaligal** [comment]
@RayBB Yes, I am using Meks template. I will be opening a PR soon. Thankyou for the feedback
